eth2near/eth2near-block-relay-rs/src/hand_made_finality_light_client_update.rs

```diff
@@ -31,82 +34,163 @@ impl HandMadeFinalityLightClientUpdate {
 
         let attested_header = beacon_rpc_client
             .get_beacon_block_header_for_block_id(&format!("{}", attested_slot))?;
+
         let beacon_state = beacon_rpc_client.get_beacon_state(&format!("{}", attested_slot))?;
+
         let finality_hash = beacon_state.finalized_checkpoint().root;
         let finality_header = beacon_rpc_client
             .get_beacon_block_header_for_block_id(&format!("{:?}", &finality_hash))?;
 
+        let finalized_block_body = beacon_rpc_client
+            .get_beacon_block_body_for_block_id(&format!("{:?}", &finality_hash))?;
+
+        let sync_committee_bits: [u8; 64] = Self::get_sync_committee_bits(sync_committe_signature)?;
+
+        Ok(LightClientUpdate {
+            attested_beacon_header: Self::from_lighthouse_beacon_header(&attested_header),
+            sync_aggregate: eth_types::eth2::SyncAggregate {
+                sync_committee_bits: SyncCommitteeBits(sync_committee_bits),
+                sync_committee_signature: serde_json::from_str::<SignatureBytes>(
+                    &serde_json::to_string(&sync_committe_signature.sync_committee_signature)?,
+                )?,
+            },
+            signature_slot,
+            finality_update: Self::get_finality_update(
+                &finality_header,
+                &beacon_state,
+                &finalized_block_body,
+            )?,
+            sync_committee_update: match include_next_sync_committee {
+                false => Option::<SyncCommitteeUpdate>::None,
+                true => Some(Self::get_next_sync_committee(finality_header.slot
+                    .as_u64(), &beacon_rpc_client)?),
```

Comment:
Wrong formatting + redundant borrow

```suggestion
                true => Some(Self::get_next_sync_committee(
                    finality_header.slot.as_u64(),
                    beacon_rpc_client,
                )?),
```